zfec/cmdline_zunfec.py

```diff
@@ -3,18 +3,24 @@
 # zfec -- a fast C implementation of Reed-Solomon erasure coding with
 # command-line, C, and Python interfaces
 
-import os, sys
+from __future__ import print_function
 
-import argparse
-import filefec
+import os, sys, argparse
+
+from zfec import filefec
 
 from zfec import __version__ as libversion
 __version__ = libversion
 
+try:
+    unicode = unicode
+except NameError:
+    unicode = str
```

Comment:
this is a bit unusual. the recommended style of a port that works on python 3 and 2 is that code should look as in python3 (or somehow close) and/or be close to python3 semantics.

that makes it easier for the future (when nobody uses 2 any more and at some time, support for 2 will get removed).

I guess most people use something like this:
```
text_type = str  # python 3
text_type = unicode  # python 2
```